### DISCUSSION TITLE
Why create new conversions even the conversation already created

### DISCUSSION THREAD

**Discussion by NitinNR** [chatwoot/chatwoot#4364] [Q&A] [answered]
If the conversation already created by some integration why there is option in that contact details that is responsible for creating the new conversation again.

why need to create multiple conversations for the same contact (user)

**Comment by pranavrajs** [comment]
@NitinNR A contact in Chatwoot can have more than one conversation on their end. For eg: if you are on email channel, you can have more than one email thread with another person, right? The model is built in a view to support multiple conversation per contact.

Is there any issue which you are facing?

**Reply by NitinNR** [reply]
But there should be only one conversation for that contact per inbox or per integration.

**Comment by sojan-official** [comment, marked as answer]
@NitinNR  The integrations have the ability to determine whether one conversation or multiple conversations should exist for a contact. This allows catering for the cases in different channels as well to support systems where each query that comes in a conversation is treated as a new support ticket.

- Twitter DM : a single conversation 
- tweets: multiple conversations
- email channel: multiple conversations
- messenger channel: a single conversation

if you are building a custom integration using an API channel you could explore both the models 
- Create a single conversation and create messages against that single conversation
- Create multiple conversations based on your requirement.

This approach gives more flexibility to the chatwoot Platform.

read more here: https://github.com/chatwoot/chatwoot/wiki/Building-on-Top-of-Chatwoot:-Importing-Existing-Contacts-and-Creating-Conversations#associations

**Reply by NitinNR** [reply]
Thanks that explains a lot
